cpp/cli_main.cc

```diff
@@ -168,6 +169,10 @@ void Chat(tvm::runtime::Module chat_mod, const std::string& model, int64_t max_g
       continue;
     } else if (inp.substr(0, 5) == "/exit") {
       break;
+    } else if (inp.substr(0, 6) == "/stats") {
```

Comment:
can we add a hint when initiating the CLI, saying what those special commands are?